javascript/tree-js/index.js

```diff
@@ -34,4 +33,64 @@ const makeTreeObj = {
   ],
 }
 
-tree(makeTreeObj, '')
+const makeTreeObjLong = {
```

Comment:
that works!
<img width="299" alt="Screenshot 2020-08-17 at 14 19 51" src="https://user-images.githubusercontent.com/1259644/90395404-cb5ef100-e094-11ea-9912-8d42349d71f0.png">